licensing/locale/fr_FR/LC_MESSAGES/messages.po

```diff
@@ -832,7 +832,7 @@ msgstr "moins..."
 
 #: ajax_htmldata.php:223 ajax_htmldata.php:865 ajax_htmldata.php:983
 msgid "(none uploaded)"
-msgstr "(non téléchargé)"
+msgstr "(aucun téléchargé)"
```

Comment:
"Téléchargé" often means downloaded.
We might consider using "Téléversé" here. ( https://fr.wiktionary.org/wiki/t%C3%A9l%C3%A9verser )